modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -638,6 +654,38 @@ private CompletableFuture<Void> waitForMetadataCompleteness(long ts) {
         return executorInclinedSchemaSyncService.waitForMetadataCompleteness(HybridTimestamp.hybridTimestamp(ts));
     }
 
+    private CompletableFuture<Boolean> onZoneReplicaCreated(PartitionReplicaEventParameters parameters) {
+        if (!PartitionReplicaLifecycleManager.ENABLED) {
+            return completedFuture(false);
+        }
+
+        List<CompletableFuture<?>> futs = new ArrayList<>();
+
+        readyToProcessTableStarts.thenRun(() -> {
+            Set<TableImpl> zoneTables = zoneTables(parameters.zoneDescriptor().id());
```

Comment:
Seems that you only use id from the descriptor. In that case, as mentioned earlier, I'd rather user replicationGroupId.

Reply:
Fixed